entity-types/infra-awsnatgateway/golden_metrics.yml

```diff
@@ -0,0 +1,70 @@
+activeConnectionCountRate:
+  query:
+    eventId: entity.guid
+    select: max(aws.natgateway.ActiveConnectionCount)
+    from: Metric
+  unit: COUNT
+  title: Number of concurrent active TCP connections through the NAT gateway
+packetsOutToSourceRate:
+  query:
+    eventId: entity.guid
+    select: rate(sum(aws.natgateway.PacketsOutToSource), 1 minute)
```

Comment:
Would you think the title should include `rate` or the fact that packets per minute?
Also, should the unit be MESSAGES_PER_SECOND or another rate one?
Apply to all the others...
Look at https://github.com/newrelic/entity-definitions/blob/main/docs/entities/golden_metrics.md#metric-unit

Reply:
AWS docs state it as COUNT, so I rather stick with it: https://docs.aws.amazon.com/vpc/latest/userguide/vpc-nat-gateway-cloudwatch.html

Reply:
About the naming, I wen with *PerMinute because looks more practical.